projects/1.20/assets/the-outer-end/outer_end/lang/zh_cn.json

```
@@ -0,0 +1,215 @@
+{
+  "block.outer_end.azure_stem": "碧蓝木原木",
+  "block.outer_end.azure_pith": "碧蓝木木髓",
+  "block.outer_end.azure_stripped_stem": "去皮碧蓝木原木",
+  "block.outer_end.azure_stripped_pith": "去皮碧蓝木木髓",
+  "block.outer_end.azure_planks": "碧蓝木木板",
+  "block.outer_end.azure_leaves": "碧蓝木树叶",
+  "block.outer_end.azure_stamen": "碧蓝木花蕊",
+  "block.outer_end.azure_bud": "碧蓝木树苗",
```

```suggestion
  "block.outer_end.azure_stem": "碧蓝原木",
  "block.outer_end.azure_pith": "碧蓝木髓",
  "block.outer_end.azure_stripped_stem": "去皮碧蓝原木",
  "block.outer_end.azure_stripped_pith": "去皮碧蓝木髓",
  "block.outer_end.azure_planks": "碧蓝木板",
  "block.outer_end.azure_leaves": "碧蓝树叶",
  "block.outer_end.azure_stamen": "碧蓝花蕊",
  "block.outer_end.azure_bud": "碧蓝幼苞",
```
建议参照原版“白桦”系列翻译和库内1165版本翻译
Stamen和Bud在游戏里是啥样的